### PR TITLE
[DNM] jewel: rgw: no http referer info in container metadata dump in swift API 

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/18897

### CODE REVIEW

**Comment by Jing-Scott** [comment]
Hi, @smithfarm it depends on the swift http reference feature, which has not been backported to jewel. So it had a compilation error

**Comment by smithfarm** [comment]
@Jing-Scott Thanks for the clarification. I unstaged this backport for now. If you hear about that feature getting backported to jewel, feel free to ping me here.

**Comment by smithfarm** [comment]
Jenkins re-test this please

File: src/rgw/rgw_acl_swift.cc [hunk 1]
```diff
@@ -119,10 +119,17 @@ void RGWAccessControlPolicy_SWIFT::to_str(string& read, string& write)
     ACLGrant& grant = iter->second;
     int perm = grant.get_permission().get_permissions();
     rgw_user id;
+    string url_spec;
```
Comment (amitkumar50) [inline src/rgw/rgw_acl_swift.cc]: @smithfarm 
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits

**Pull request closed without merging**